scripts/bootstrap.sh.in

```diff
@@ -55,11 +55,20 @@ BASE_DIR=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
 
 SALT_CALL=${SALT_CALL:-salt-call}
 
-declare -a PACKAGES=(
+declare -a PACKAGES_COMMON=(
     salt-minion-@@SALT_VERSION
     genisoimage
+)
+# shellcheck disable=SC2034
+# Used through an indirection
+declare -a PACKAGES_REDHAT_7=(
     python36-rpm
```

Comment:
Whenever I see `python36-rpm` for EL7 and `python36` for EL8 I'm very confused. `python36-rpm` is not alike `python36`, right?

Reply:
No, python36-rpm is the python36 lib to interact with rpm
Maybe it's not needed in rhel8 (and btw it's maybe also no longer needed with salt 3002.2 we have now, I didn't checked, but it's not related to this PR btw https://github.com/saltstack/salt/issues/57972)
I agree it's really different stuff and to be consistent we may want to install "python36-rpm" on rhel8 as well (if this package exists, otherwise maybe we should add also "python36" in the list for rhel7 even if it's not really needed since it's a dependency of "python36-rpm")

Reply:
Exactly. So, if we need `python36-rpm` on RHEL7 systems *and* `python36` (when not using the RPM bindings), we should declare `python36` as a dependency, and not rely on it being implicitly available because we install `python36-rpm` which has `python36` as a dep.
If on RHEL8 we *don't* need `python36-rpm`, then it shouldn't be listed and only include `python36` (or similar).

Let's try to keep the differences between the OSes as minimal as possible.